app/schemas/models/podcast.schema.js

```diff
@@ -0,0 +1,24 @@
+const schema = require('./../schemas')
+
+const PodcastSchema = schema.object(
+  {
+    title: 'Podcast Episode',
+    required: ['name']
+  },
+  {
+    name: schema.shortString(),
+    description: { type: 'string' },
+    guestName: schema.shortString(),
+    guestDetails: { type: 'string' },
+    guestImage: { type: 'string', format: 'image-file' },
+    transcriptUrl: schema.url(),
+    uploadDate: schema.date(),
+    transistorEpisodeId: schema.shortString(),
+    audio: schema.sound()
```

Comment:
Also need to specify which fields are translated, like [here](https://github.com/codecombat/codecombat/blob/master/app/schemas/models/resource_hub_resource.schema.js#L31-L35). I think this is `['name', 'description', 'guestDetails', 'transcriptUrl']`.